bundles/org.openhab.binding.dominoswiss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dominoswiss"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="egate">
+		<label>Dominoswiss EGate Server</label>
+		<description>This is a Dominoswiss EGate Server instance.</description>
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>IP or Host Name</label>
+				<context>network-address</context>
+				<description>The IP or host name of the Dominoswiss EGate Server (192.168.1.100, localhost)</description>
+			</parameter>
+			<parameter name="port" type="integer" required="true" min="1" max="65335">
+				<label>Web Port</label>
+				<description>Port interface of the Dominoswiss EGate Server, default 1318</description>
+				<default>1318</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="blind">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="egate"/>
+		</supported-bridge-type-refs>
+		<label>Blinds controlled by Dominoswiss</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
		<label>Blind</label>
```